package.json

```diff
@@ -45,6 +45,7 @@
     "downshift": "3.2.10",
     "emotion": "9.2.12",
     "emotion-theming": "9.2.9",
+    "faker": "^4.1.0",
```

Comment:
What does faker do? We only use it in one place.

Reply:
I see, it's to create fake data.

Reply:
we don't necessarily need that in ui-kit and i can remove that for sure before merging unless @mperrotti or @Poltergeist think it would be worthwhile to have that in.

i thought it might be cool to have lots of different data an then be forced to write content-agnostic tests.

Reply:
I think it's better to remove it before merging.

Reply:
I think it might be nice to keep it in for testing purposes, but it's not totally necessary.